displaynames.html

```diff
@@ -149,21 +149,19 @@ <h1>Intl.DisplayNames ( _locales_, _options_ )</h1>
         1. Set _displayNames_.[[Locale]] to the value of _r_.[[Locale]].
         1. Let _dataLocale_ be _r_.[[dataLocale]].
         1. Let _dataLocaleData_ be _localeData_.[[&lt;_dataLocale_&gt;]].
-        1. <del>Let _types_ be _dataLocaleData_.[[types]].</del>
+        1. Let _types_ be _dataLocaleData_.[[types]].
+        1. Assert: _types_ is a Record (see <emu-xref href="#sec-Intl.DisplayNames-internal-slots"></emu-xref>).
+        1. Let _typeFields_ be _types_.[[&lt;_type_&gt;]].
+        1. Assert: _typeFields_ is a Record (see <emu-xref href="#sec-Intl.DisplayNames-internal-slots"></emu-xref>).
+        1. <del>Let _styleFields_ be _typeFields_.[[&lt;_style_&gt;]].</del>
         1. <ins>If _type_ is `"language"`, then</ins>
           1. <ins>Let _dialectHandling_ be ? GetOption(_options_, `"dialectHandling"`, `"string"`, &laquo; `"dialectName"`, `"standardName"` &raquo;, `"dialectName"`).</ins>
           1. <ins>Set _displayNames_.[[DialectHandling]] to _dialectHandling_.</ins>
-          1. <ins>Let _dialects_ be _dataLocaleData_.[[dialects]].</ins>
-          1. <ins>Assert: _dialects_ is a Record (see <emu-xref href="#sec-Intl.DisplayNames-internal-slots"></emu-xref>).</ins>
-          1. <ins>Let _dialectFields_ be _dialects_.[[&lt;_dialectHandling_&gt;]].</ins>
+          1. <ins>Let _dialectFields_ be _typeFields_.[[&lt;_dialectHandling_&gt;]].</ins>
           1. <ins>Assert: _dialectFields_ is a Record (see <emu-xref href="#sec-Intl.DisplayNames-internal-slots"></emu-xref>).</ins>
-          1. <ins>Let _types_ be _dialectFields_.[[&lt;_type_&gt;]].</ins>
+          1. <ins>Let _styleFields_ be _dialectFields_.[[&lt;_style_&gt;]].</ins>
         1. <ins>Else,</ins>
-          1. <ins>Let _types_ be _dataLocaleData_.[[types]].</ins>
-        1. Assert: _types_ is a Record (see <emu-xref href="#sec-Intl.DisplayNames-internal-slots"></emu-xref>).
-        1. Let _typeFields_ be _types_.[[&lt;_type_&gt;]].
-        1. Assert: _typeFields_ is a Record (see <emu-xref href="#sec-Intl.DisplayNames-internal-slots"></emu-xref>).
-        1. Let _styleFields_ be _typeFields_.[[&lt;_style_&gt;]].
+          1. <ins>Let _styleFields_ be _typeFields_.[[&lt;_style_&gt;]].</ins>
```

Comment:
Minor editorial nit, but the `Else` can be avoided here by setting to the value of `typeFields` again...

Reply:
PTAL